### ISSUE TITLE
no import LOGGER, get error

### ISSUE THREAD

**Issue by PotekhinRoman** [JaidedAI/EasyOCR#273]
no import LOGER here, we get an error if the image is None: 
https://github.com/JaidedAI/EasyOCR/blob/8af936ba1b2f3c230968dc1022d0cd3e9ca1efbb/easyocr/utils.py#L627

test:
`
import cv2
import easyocr

path_to_image = 'no_image_here'
image = cv2.imread(path_to_image)
reader = easyocr.Reader(lang_list=['en'])
reader.detect(image)
`
result:
"name 'LOGGER' is not defined"

**Comment by gregd33** [comment]
I would also add that if the code gets to that point (of trying to log a warning) it will break.

It will try to return `img` and `img_cv_grey` which are undefined so you'll get `local variable 'img' referenced before assignment`. And there isn't anything sensible to return either... so maybe better to raise an exception?

**Comment by ShaheenPerveen** [comment]
facing the same issue:
NameError: name 'LOGGER' is not defined

**Comment by volkov-maxim** [comment]
Issue solved with correct path to filename:

reader.readtext('..\test\filename.jpg') - raise error
reader.readtext(r'..\test\filename.jpg') - do right thing

**Comment by rkcosmos** [comment]
yes, this error is caused by bad image path. I'll modify the code to produce more informative error message.
